docs/explanations/components/property_package/general_reactions/generic_reactions.rst

```diff
@@ -27,7 +27,7 @@ The most common way to use the Generic Reaction Package Framework is to create a
 
     m.fs.thermo_properties = PhysicalParameterBlock()
 
-    m.fs.reaction_properties = GenericReactionParameterBlock(default={"property_package": m.fs.thermo_properties, config_dict})
+    m.fs.reaction_properties = GenericReactionParameterBlock(property_package=m.fs.thermo_properties, **config_dict)
```

Comment:
Why were the asterisks not needed before? Is this changed required now?

Reply:
The dictionary ``config_dict`` was passed to a keyword argument called ``default`` as a dictionary.  Now we are pass in the same information directly as keyword args.  The asterisks expand the dict into keyword arguments.